src/test/java/com/wikia/webdriver/testcases/mercurytests/CategoryPageTest.java

```diff
@@ -0,0 +1,54 @@
+package com.wikia.webdriver.testcases.mercurytests;
+
+import com.wikia.webdriver.common.contentpatterns.MercuryWikis;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.pages.ArticlePage;
+import com.wikia.webdriver.elements.mercury.pages.CategoryPage;
+
+import org.testng.annotations.Test;
+
+@Execute(onWikia = MercuryWikis.MERCURY_AUTOMATION_TESTING)
+@InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+public class CategoryPageTest extends NewTestTemplate {
+
+  @Test(groups = "mercury_category_navigateToCategoryPageWithoutArticleAndWithMembersFromUrl")
+  public void mercury_category_navigateToCategoryPageWithoutArticleAndWithMembersFromUrl() {
+    new CategoryPage().navigateToPageWithoutArticleAndWithMembersFromUrl();
+  }
+
+  @Test(groups = "mercury_category_navigateToCategoryPageWithArticleAndWithMembersFromUrl")
+  public void mercury_category_navigateToCategoryPageWithArticleAndWithMembersFromUrl() {
+    new CategoryPage().navigateToPageWithArticleAndWithMembersFromUrl();
+  }
+
+  @Test(groups = "mercury_category_navigateToCategoryPageWithArticleAndWithoutMembersFromUrl")
+  public void mercury_category_navigateToCategoryPageWithArticleAndWithoutMembersFromUrl() {
+    new CategoryPage().navigateToPageWithArticleAndWithoutMembersFromUrl();
+  }
+
+  @Test(groups = "mercury_category_navigateToCategoryPageWithArticleAndWithoutMembersFromLink")
+  public void mercury_category_navigateToCategoryPageFromLinkInArticle() {
+    new ArticlePage()
+        .openPageWithLinkToCategoryPage()
+        .openLinkToCategoryPage();
+  }
+
+  @Test()
+  public void mercury_category_new() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)